src/styles/theme/oc-dropzone.scss

```
@@ -1,17 +1,16 @@
 .oc-dropzone {
-  @extend .uk-position-fixed;
+  @extend .uk-position-cover;
+  @extend .uk-text-center;
   @extend .uk-flex;
   @extend .uk-flex-center;
   @extend .uk-flex-middle;
   @extend .uk-position-z-index;
 
   background-color: rgba($global-background, 0.8);
-  height: calc(100% - (2 * #{$navbar-nav-item-height}));
-  left: 0;
-  top: 2 * $navbar-nav-item-height;
-  width: 100%;
 }
 
 .oc-dropzone span {
   pointer-events: none;
 }
+switch oc-dropzone positioning from fixed to absolute, fixed always orientates by the viewport which means it always coveres the entire screen. This is something we could't know and the decision should be left to the consuming app. instead we use position absolute to just cover the next parent which ist positioned relative or the viewport.
+in situations where the body is higher or wider then the viewport fixed works better because absolute won't cover the scrollable parts of the content. since oc apps always do scrolling on their own this is no problem
```

```suggestion
In situations where the body is higher or wider then the viewport fixed works better because absolute won't cover the scrollable parts of the content. Since oc apps always do scrolling on their own this is no problem
```

Hmm - are the lines supposed to be added to this file?